catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -272,136 +281,160 @@ public static DAG convertResult(Result result, ORB orb, POA poa, List<String> re
+                if (pocAttr != null) {
+                    String pocString = String.valueOf(pocAttr.getValue());
+                    if (StringUtils.isNotBlank(pocString)) {
+                        addStringAttribute(graph, fileNode, NsiliConstants.CREATOR, pocString, orb);
+                    } else {
+                        addStringAttribute(graph,
+                                fileNode,
+                                NsiliConstants.CREATOR,
+                                SystemInfo.getSiteName(),
+                                orb);
+                    }
+                } else {
                     addStringAttribute(graph,
                             fileNode,
-                            NsiliConstants.PRODUCT_URL,
-                            downloadUrl,
+                            NsiliConstants.CREATOR,
+                            SystemInfo.getSiteName(),
                             orb);
-                    addedAttributes.add(buildAttr(attribute, NsiliConstants.PRODUCT_URL));
                 }
+                addedAttributes.add(buildAttr(attribute, NsiliConstants.CREATOR));
             }
-        }
-
-        if (shouldAdd(buildAttr(attribute, NsiliConstants.TITLE), resultAttributes)
-                && metacard.getTitle() != null) {
-            addStringAttribute(graph, fileNode, NsiliConstants.TITLE, metacard.getTitle(), orb);
-            addedAttributes.add(buildAttr(attribute, NsiliConstants.TITLE));
-        }
-
-        if (shouldAdd(buildAttr(attribute, NsiliConstants.IS_PRODUCT_LOCAL), resultAttributes)) {
-            String siteName = SystemInfo.getSiteName();
-            boolean productLocal = true;
-            if (siteName != null && metacard.getSourceId() != null
-                    && !siteName.equals(metacard.getSourceId())) {
-                productLocal = false;
-            }
-            addBooleanAttribute(graph,
-                    fileNode,
-                    NsiliConstants.IS_PRODUCT_LOCAL,
-                    productLocal,
-                    orb);
-            addedAttributes.add(buildAttr(attribute, NsiliConstants.IS_PRODUCT_LOCAL));
-        }
 
-        if (shouldAdd(buildAttr(attribute, NsiliConstants.DATE_TIME_DECLARED), resultAttributes)) {
-            if (metacard.getCreatedDate() != null) {
-                addDateAttribute(graph,
-                        fileNode,
-                        NsiliConstants.DATE_TIME_DECLARED,
-                        metacard.getCreatedDate(),
-                        orb);
-            } else {
-                addDateAttribute(graph,
+            if (shouldAdd(buildAttr(attribute, NsiliConstants.IS_PRODUCT_LOCAL),
+                    resultAttributes)) {
+                String siteName = SystemInfo.getSiteName();
+                boolean productLocal = true;
+                if (siteName != null && metacard.getSourceId() != null
+                        && !siteName.equals(metacard.getSourceId())) {
```

Comment:
We don't do anything different whether the product is local or not. I think this is mainly used by 4559 implementers when they are synchronizing to know whether or not they have cached product data locally.